lib/order.rb

```diff
@@ -0,0 +1,61 @@
+require 'csv'
+require_relative 'customer'
+
+class Order 
+    attr_reader :id 
+    attr_accessor :products, :customer, :fulfillment_status 
+    
+    
+    def initialize (id, products, customer, fulfillment_status = :pending)
+        @id = id 
+        @products = products 
+        @fulfillment_status = fulfillment_status
+        @customer = customer 
+        
+        fulfillment_statuses = [:pending, :paid, :processing, :shipped, :complete ]
+        if !fulfillment_statuses.include?(@fulfillment_status)
+            raise ArgumentError 
```

Comment:
It would be helpful to include a message here so the user knows what went wrong.

```suggestion
            raise ArgumentError.new("Invalid fulfillment status: #{@fulfillment_status}")
```